Try tests on php 5.6 without the ini_set workarounds

greg-1-anderson opened the pull request:
To see if the APM changes fixed this problem.

greg-1-anderson committed:
Try tests on php 5.6 without the ini_set workarounds, to see if the APM changes fixed this problem.

pantheon-circleci commented:
@greg-1-anderson, thanks for your PR! By analyzing the blame information on this pull request, we identified @danielbachhuber and @stevector to be potential reviewers

greg-1-anderson committed:
Put the ini_set commands back

Tests failed; putting the ini_set back to see if tests pass as-is with php 5.6.

greg-1-anderson commented:
@danielbachhuber: Any idea why these tests are failing under php 5.6?  WordPress works on 5.6 and 7.0, afaik.

danielbachhuber commented on prepare.sh:
```diff
@@ -12,6 +12,7 @@ fi
 ###
 terminus site create-env --to-env=$TERMINUS_ENV --from-env=dev
 yes | terminus site wipe
+terminus site set-php-version --version=5.6
```
This should have `--env=$TERMINUS_ENV`, otherwise it sets the site default.

greg-1-anderson replied:
Whoops, sorry about the typo.

greg-1-anderson committed:
Explicitly pass --env to terminus site set-php-version

Otherwise, it will set the default php version

greg-1-anderson commented:
It seems there are still error handling problems with PHP 5.6 and 7.0 on Pantheon when running phpunit, and the old workaround is no longer sufficient.

Commit:
merging updates from master

Commit:
use pantheon.yml to set PHP version

Commit:
Specify PHP 5.6 properly
